homeassistant/components/sensor/arest.py

```diff
@@ -52,9 +52,11 @@ def make_renderer(value_template):
         if value_template is None:
             return lambda value: value
 
+        value_template = template.Template(value_template, hass)
```

Comment:
This will need updating if https://github.com/home-assistant/home-assistant/pull/3487 gets merged before this one.